diff --git a/node_modules/101/set.js b/node_modules/101/set.js
index 0a8178a..56d929a 100644
--- a/node_modules/101/set.js
+++ b/node_modules/101/set.js
@@ -19,6 +19,9 @@ var keypather = require('keypather')();
 module.exports = set;
 
 function set (obj, key, val) {
+  if (key === '__proto__' || key === 'prototype' || key === 'constructor') {
+    return
+  }
   var setObj;
   if (arguments.length === 1) {
     // (setObj)